dev-python/authlib/metadata.xml

```diff
@@ -0,0 +1,23 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE pkgmetadata SYSTEM "https://www.gentoo.org/dtd/metadata.dtd">
+<pkgmetadata>
+	<maintainer type="person" proxied="yes">
+		<name>Oz Tiram</name>
+		<email>oz.tiram@gmail.com</email>
+	</maintainer>
+	<maintainer type="project" proxied="no">
+		<name>Proxy Maintainers</name>
+		<email>proxy-maint@gentoo.org</email>
+	</maintainer>
+	<use>
+		<flag name="django">Enable Django framework support</flag>
+		<flag name="flask">Enable Flask framework support</flag>
+		<flag name="jose">Enable JSON Object Signing and Encryption (JOSE) support via <pkg>dev-python/pycryptodome</pkg></flag>
+	</use>
+	<upstream>
+		<remote-id type="github">lepture/authlib</remote-id>
+		<remote-id type="pypi">Authlib</remote-id>
+		<doc>https://docs.authlib.org/</doc>
+		<bugs-to>https://github.com/authlib/authlib/issues</bugs-to>
```

Comment:
Doesn't match remote-id.

Reply:
The ebuild now has both. Should both have the same order?

Reply:
I don't understand.

Reply:
Sorry, I misunderstood your original comment. I think you meant to say I am missing pypi in HOMEPAGE in the ebuild.

Reply:
No, I mean that remote-id is `lepture/authlib` and this URL is `authlib/authlib`.